### ISSUE TITLE
Pressing 'START' then cancel - will not cancel

### ISSUE THREAD

**Issue by AlonDan** [s0md3v/roop#436]
**Pressing the START button won't cancel**
Once pressing START (browsing to select OUTPUT filename) if pressing CACNEL to regret..
It will close the browse window but will start processing anyway, so it's not actually canceling.

**To Reproduce**
1. Load your Source Image and Target Video
2. START
3. While browsing to select the path and output filename press CANCEL
4. It will start process anyway instead of cancel

**Details**
What OS are you using?
- [x] Windows 

Are you try to use a GPU?
- [x] NVIDIA

**Sanity Check**
- [x] I have the latest code from the github repository
- [x] I have followed the installation guide

**Comment by henryruhs** [comment]
There is no cancel button present. Just destroy and this actually does what it says.

**Comment by AlonDan** [comment]
Sorry about my bad English maybe I wasn't clear, let me try again:

On windows when you browse to select the output path and file name:
that **browse window** have OPEN or SELECT options, if clicked select, it will still process same as pressed OPEN, obviously it's wrong.

**Comment by henryruhs** [comment]
Try the next branch, issue should be solved there.
